layer4/matchers.go

```diff
@@ -166,3 +151,26 @@ var (
 	_ ConnMatcher       = (*MatchIP)(nil)
 	_ caddy.Provisioner = (*MatchIP)(nil)
 )
+
+func ParseNetworks(networks []string) (ipNets []*net.IPNet, err error) {
```

Comment:
Let's add a godoc comment here explaining what this does

Reply:
Done